common-lib/src/main/scala/com/gu/mediaservice/lib/cleanup/SupplierProcessors.scala

```diff
@@ -275,6 +287,11 @@ trait GettyProcessor {
 }
 
 object GettyXmpParser extends ImageProcessor with GettyProcessor {
+  def getSuppliersReference(image: Image)={
+    // String defined in ImageMetadataConvertor
+    image.fileMetadata.getty.get("Asset ID").orElse(image.metadata.suppliersReference)
```

Comment:
Let’s not read that from our copy of Getty data (even though it conveniently deals with inconsistent naming), because we should (and will) get rid of this stupid copy anyway. It was introduced when we weren’t yet reading the whole of XMP. Getty processor needs to move to identifying Getty images by existence of Asset ID, not, like now, by the existence of Getty namespace only (we will be able to remove all exclusions then, yay!).

Instead, let’s read it from main XMP block. If we are feeling generous, we could include naming reconcilliation of old, rare spelling (I think the old one is `AssetId` and correct, more common, one is `AssetID`).

Reply:
spoil sport

Reply:
Looking at it, we still use that code in the Getty Cleaner- and it's a bit of a faff to read it  from JsValue.